core/src/plugins/container/cloudbuilder.ts

```
@@ -435,12 +435,21 @@ class BuildxBuilder {
   }
 
   private async installDirectly() {
-    const statResult = await stat(dirname(this.buildxInstanceJsonPath))
-    if (statResult.isDirectory()) {
-      await writeFile(this.buildxInstanceJsonPath, JSON.stringify(this.getBuildxInstanceJson()))
-      return true
+    try {
+      const statResult = await stat(dirname(this.buildxInstanceJsonPath))
+      if (statResult.isDirectory()) {
+        await writeFile(this.buildxInstanceJsonPath, JSON.stringify(this.getBuildxInstanceJson()))
+        return true
+      }
+      return false
+    } catch (e) {
+      // An error is thrown e.g. if the path does not exist.
+      // We don't need to handle this error, as we will fall back to the CLI installation.
+      if (isErrnoException(e) && e.code === "ENOENT") {
+        return false
```

Should we still log the error though? In case there are other failure modes we're not thinking of which we won't see if the error is swallowed.